test/manual_test_filter_config_widgets.py

```diff
@@ -2,9 +2,8 @@
 from PySide6.QtWidgets import QMainWindow, QVBoxLayout
```

Comment:
encoding und doku

Reply:
I cannot document this file as I have not written it nor touched it in a manual manner. However the effects stacking system is something that would be a perfect fit for unit tests. We should write some for it once we get to it.